comp110_architecture.tex

```diff
@@ -5,41 +5,150 @@
 \usepackage[hidelinks]{hyperref}
 \usepackage[none]{hyphenat}
 \usepackage{setspace}
+\usepackage{graphicx}
+\graphicspath{ {G:/GitHubGames/comp110-architecture-essay/Pictures/} }
+	
 \doublespace
 
 % You may add additional packages here
 \usepackage{amsmath}
 
 % Please include a clear, concise, and descriptive title
-\title{Your Title Here}
+\title{Implementation of PCG: Three Examples But Which Is The Most Practical?}
 
 % Please do not change the subtitle
 \subtitle{COMP110 - Computer Architecture Essay}
 
 % Please put your student number in the author field
-\author{DO NOT WRITE YOUR NAME\\Put your student number (on your ID card) here}
+\author{1506530}
 
 \begin{document}
 
 \maketitle
 
-\abstract{Please include an abstract of at most 100 words (these do not count towards your word count).}
+\abstract{Procedural content generation (PCG) has been used in games for decades now. It is an ever evolving method for creating emersive and reliable game that players can enjoy. This paper will provide three distinct methods for implementing PCG in a 2D platfrom game. They will be of an increasing implementation ease, starting from a relatively simple algorithm that is used to enhance the difficulty of a game. Then moving on to one that uses an already existing game to randomize its levels. Finally creating a complete new game based on PCG.}
```

Comment:
Avoid colloquial tones (e.g. "for decades now"). Try to follow an OCAR structure for the abstract --- this will become easier to do once you have finished the rest of the essay.